File: BasitIslemler.swift
//  Created by Kağan on 15.01.2017.
//  Copyright © 2017 Kağan Utku Kılıçlı. All rights reserved.

import Foundation


print("lütfen birinci sayiyi giriniz:")
if var sayiA = Double(readLine(strippingNewline: true)!) {
    print("lütfen ikinci sayiyi giriniz:")
    if var sayiB = Double(readLine(strippingNewline: true)!) {
        print("toplam: \t \(sayiA + sayiB)")
        print("fark: \t \(sayiA - sayiB)")
        print("carpim: \t \(sayiA * sayiB)")
        print("bolum: \t \(sayiA / sayiB)")
        print("kalan: \t \(Int(sayiA) % Int(sayiB))")
    }
}

File: BasitIslemler.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);

        int toplam;
        int fark;
        int carpim;
        double bolum;
        int kalan;

        System.out.print("Lütfen birinci sayıyı girin : ");
        int number1 = scanner.nextInt();
        System.out.print("Lütfen ikinci sayıyı girin : ");
        int number2 = scanner.nextInt();

        toplam = (number1 + number2);
        fark = (number1 - number2);
        carpim = (number1 * number2);
        bolum = ((double) number1 / number2);
        kalan = (number1 % number2);

        System.out.println("Toplam : " + toplam);
        System.out.println("Fark : " + fark);
        System.out.println("Çarpım : " + carpim);
        System.out.println("Bölüm : " + bolum);
        System.out.println("Kalan : " + kalan);

    }
}
